content/en/docs/reference/issues-security/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - committee-security-response
+  - sig-security-leads
+
+reviewers:
+  - committee-security-response
+  - sig-security-leads
+
+labels:
+  - sig/security
+  - sig/docs
```

Comment:
nit: How are the labels used for this group of owners (specifically the sig/docs label)?

Reply:
@kbhawkey thank you for the comment. I am struggling to understand though, unfortunately. Can you please elaborate a bit more?

Reply:
I am not clear how you plan to use the labels?

Reply:
@PushkarJ : Please could you advise?

Reply:
@divya-mohan0209 Sorry I took a while to respond. I can honestly remove the labels if it does not serve any purpose. Would that suffice to resolve your comment @kbhawkey